### ISSUE TITLE
Document excess data movement when decommissioning nodes one at a time

### ISSUE THREAD

**Issue by rmloveland** [cockroachdb/docs#5078]
Richard Loveland (rmloveland) commented:

tl;dr: If you want to decommission N nodes, and you do them one at a
time, you incur unnecessary data movement costs.

Quoth @tbg:

> Say you are decommissioning first node 1, then node 2, then
> node 3. Some of the replicas from node 1 will move to nodes 2 and 3.
> Then you decommission 2, so now some replicas that just got moved on
> 2 have to move again; some of them will end up on 3. The game
> repeats at 3. This will definitely prolong the complete
> decommissioning since a lot of unnecessary work is being done.  It
> should be perfectly fine to decommission multiple nodes at once

The docs mention both the "one at a time" and "N at a time" cases
separately, but their current structure emphasizes the single-node
case.  They also don't mention the "excessive data movement" info Tobi
supplied above.

Estimated scope of work on this ticket:

- Note that single-node decommissioning will incur unnecessary data
  movement costs if you are doing them serially for some reason when
  what you really want is to decommission some number N > 1 all at
  once

- Restructure to make it clearer that the single-node case is not
  somehow preferred or privileged despite its larger word count /
  greater # of examples

Jira Issue: DOC-294

**Comment by rmloveland** [comment]
FYI @lhirata

**Comment by tbg** [comment]
There's some more discussion on the original issue which boils down to "can
we just play it safe still? Our testing isn't great and while it *should*
work we may want to work more on confirming this".

Our testing should be better by the end of this cycle thanks to @knz but
it's unclear whether we'll be looking at decommissioning multiple nodes
much (the infra to do so later will be there though).

**Comment by jseldess** [comment]
@taroface, after your recent docs work on decommissioning, do you think there's more to do based on this ticket?
